wdl/transforms/draft3/src/test/scala/wdl/draft3/transforms/wdlom2wom/WdlFileToWomSpec.scala

```diff
@@ -134,4 +136,21 @@ class WdlFileToWomSpec extends FlatSpec with Matchers {
         nioFileTask.inputs.find(_.name == "h").get.parameterMeta should be(Some(MetaValueElementObject(Map("localization_optional" -> MetaValueElementBoolean(true)))))
     }
   }
+
+  private def validateSameNamedInputsPriority(b: WomBundle): Assertion = {
+    val nodes = b.primaryCallable.get.asInstanceOf[WorkflowDefinition].graph.nodes
+
+    val exposedExpressionNode = nodes.filter(_.isInstanceOf[ExposedExpressionNode]).head
+    val callInputs = nodes.filter(_.isInstanceOf[TaskCallInputExpressionNode]).toList
+
+    callInputs.size shouldBe 4
+
+    // Usually, we use node names to check whether in-memory graphs are linked together correctly.
+    // Obviously, in this regression test for a same-named-node bug that won't work - instead,
+    // we use identity (the hash code) to differentiate multiple nodes with the same name.
+    callInputs(0).inputPorts.head.upstream eq exposedExpressionNode.outputPorts.head shouldBe true
```

Comment:
I'm surprised/disappointed that there's not a scalatest primitive for reference equality checks

Reply:
Upgraded to `theSameInstanceAs`, just took some digging